index.html

```diff
@@ -1,427 +1,616 @@
+                  name="full-name"
+                  placeholder="   First name"
+                  maxlength="30"
+                  id="fullNameDesk"
+                  required
+                />
+              </label>
+                                        <label for="lastName" class="lastx">
+                <input
+                  type="text"
+                  name="last-name"
+                  placeholder="  Last name"
+                  maxlength="30"
+                  id="lastName"
+                  required
+                />
+              </label>
+                                    </li>
+                                    <li class="for">
+                                        <label for="emaildesk" class="eml">
+                <input
+                  type="email"
+                  name="email"
+                  id="emaildesk"
+                  placeholder="Email address"
+                  required
+                />
+              </label>
+                                    </li>
+                                    <li class="textare">
+                                        <label for="messagedesk">
+                <textarea
+                  name="message"
+                  id="messagedesk"
+                  cols="30"
+                  rows="6"
+                  maxlength="500"
+                  placeholder="Enter text here"
+                  required
+                ></textarea>
+              </label>
+                                    </li>
+                                    <li class="buttonsub">
+                                        <button type="submit" class="button_form">Get in touch</button>
+                                    </li>
+                                </ul>
+                            </form>
+                    </div>
+                    <span><img class="spanix1" src="Pictures/linex.svg" alt="line" /></span>
+
+                    <ul class="iconx icongroup">
+                        <li class="github iconw1">
+                            <img src="Pictures/githreal.svg" alt="Github" />
+                        </li>
+
+                        <li class="linkedin linke">
+                            <img src="Pictures/linkedreal.svg" alt="linkedin" />
+                        </li>
+
+                        <li class="angel-list iconw3">
+                            <img src="Pictures/angel.svg" alt="angel-list" />
+                        </li>
+
+                        <li class="twitter iconw4">
+                            <img src="Pictures/twittereal.svg" alt="twitter" />
+                        </li>
+
+                        <li class="medium iconw5">
+                            <img src="Pictures/mediumreal.svg" alt="medium.com" />
+                        </li>
+                    </ul>
+                    </footer>
+                    <!--desktop-->
+                    <script src="script.js"></script>
+</body>
```

Comment:
- [ ] Let's try to organize code in this section of your HTML document. Taking care of indentation will help you come back later to your code and understand it. And it's even more important once you're working with others:
```
 Be careful with blank lines and indentation.
    - Do not add blank lines, spaces, or indentations without a reason.
    - For readability, add blank lines to separate large or logical code blocks.
    - For readability, add two spaces of indentation. Do not use the tab key.
      - on some code editors, you can set the tab button to do an `X` amount of spaces.
```
[You can read more about HTML & CSS best practices here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)